### PR TITLE
[TW-489] Import API from Azure Gateway

### PR DESCRIPTION
**Pull request by mark-anthony-postman**

### CODE REVIEW

**Review by HeatherStenson — APPROVED** [review]
Looks great! 🍊

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/designing-and-developing-your-api/importing-an-api.md [hunk 1]
```diff
@@ -132,7 +133,7 @@ Copy the **Role ARN** from AWS and paste it in Postman under **Step 2: Enter rol
 
 To set up a connection to Amazon API Gateway using an AWS access key, select **Access Key** under **AWS Authentication Mechanism**.
 
-<img alt="Connecting with AWS access key" src="https://assets.postman.com/postman-docs/import-from-api-gateway-with-access-key.jpg" />
+<img alt="Connecting with AWS access key" src="https://assets.postman.com/postman-docs/import-from-api-gateway-with-access-key.jpg" width="483" />
```
Comment (HeatherStenson) [inline src/pages/docs/designing-and-developing-your-api/importing-an-api.md:136]:
```suggestion
<img alt="Connecting with AWS access key" src="https://assets.postman.com/postman-docs/import-from-api-gateway-with-access-key.jpg" width="483px" />
```

**Review by HeatherStenson — APPROVED** [review]
LGTM!

File: src/pages/docs/designing-and-developing-your-api/importing-an-api.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/importing-an-api.md:136]: Thanks for catching that!

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Pull request merged**